docs/user/slicing.rst

```diff
@@ -86,6 +86,30 @@ Note: if the time slice interval is a non-integer multiple of the period, then t
         "timeSlicePeriod": 8.0,
     }
 
+Example 4: Time slicing and event filtering during loading
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+The time slicing parameters are applied relative to the whole data, independent of any event
+filtering during loading (i.e. using the parameter ``"loadOptions"``).
+
+.. code-block:: json
+
+    {
+        "sample": {
+            "loadOptions": {"FilterByTimeStart": 2.0, "FilterByTimeStop": 8.0}
```

Comment:
```suggestion
            "loadOptions": {"FilterByTimeStart": 2.0, "FilterByTimeStop": 8.71}
```
makes sure that the numerical values for "FilterByTimeStart" are "FilterByTimeStop": 8.7 are unrelated to the periodic-slicing ones